### PR TITLE
update make rest spec

### PR DESCRIPTION
**Pull request by morningvera**
We need to add this script to pull and format the full open api spec correctly. I changed the diff checker a while back but forgot to update this make command.

### CODE REVIEW

File: .polygon/rest.go [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+		body, err = io.ReadAll(resp.Body)
+		if err != nil {
+			log.Fatal(err)
+		}
+	} else {
+		log.Fatal("expected arg to be a valid URI")
+	}
+
+	var out bytes.Buffer
+	err := json.Indent(&out, body, "", "  ")
```
Comment (clickingbuttons) [inline .polygon/rest.go:36]: Can a Python script to do this instead? Go feels like a rather heavy-weight dependency to HTTP Get + format JSON.

Comment (morningvera) [reply .polygon/rest.go:36]: yeah, good point. i think it made sense in the go client but feels weird here. i'll refactor.

Comment (clickingbuttons) [reply .polygon/rest.go:36]: Cool. I think something like this is close:
```
import urllib.request
import json

contents = urllib.request.urlopen("http://example.com/foo/bar").read()
parsed = json.loads(contents)
formatted = json.dumps(parsed, indent=2)
```

Comment (morningvera) [reply .polygon/rest.go]: done!

**Review by clickingbuttons — APPROVED** [review]
Awesome, ty!!!

**Pull request merged**